FRPs/proposal-2-liquidations.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 2
+title: MEV and Liquidations
+team: Kaihua Qin, Liyi Zhou, Arthur Gervais
+created: 2020-12-10
+paper: 1
+---
+
+# Summary
+With over 6B USD total value locked for the top 3 lending protocols alone, this use case has grown into a substantial DeFi market. Lending entails the risks of a diminishing collateral value, which may render debt unhealthy. Unhealthy debt is typically resolved through liquidations, a process of selling collateral at a discount to liquidators.
+
+One dominant liquidation design is the fixed spread liquidation mechanism, used by Aave, Compound and dYdX. Interestingly, the fraction of the collateral that a liquidator can purchase at a discount amounts to 50% for Aave and Compound, and even 100% for dYdX (commonly referred to as close factor). It therefore appears that the fixed spread liquidation mechanisms are benefitting the liquidators more than the borrowers.
+
+Drawing the link to MEV, each liquidation event may qualify as Miner Extractable Value (MEV). For example, in a recent price oracle manipulation attack, a liquidator performed an oracle price post and liquidated millions of USD within one transaction, yielding a profit of nearly 4M USD. 
+
+The goal of this FRP is to offer a systematization of existing liquidation mechanisms and consistent measurements of the past and current DeFi liquidation state. We aim to investigate alternative liquidation designs in an effort to mitigate MEV from liquidation events.
+
+## Background and Problem Statement
+
+The goal of this FRP is to study the past and current status of blockchain liquidation. Specifically, we would like to answer the following questions:
+- What is the monthly accumulated liquidation profit for Aave, Compound, dYdX, MakerDAO?
+- What gas price distributions are being paid for the liquidation events?
+- What are the different liquidation methods, and how do they differ with respect to MEV? Which liquidation mechanism avoids MEV?
+- How sensitive are current debts to liquidation (by how much would an adversary need to manipulate a price oracle)? An MEV miner could try to manipulate an oracle to extract this MEV.
+- How can we design a possible liquidation mechanism that mitigates MEV?
+
+## Plan and Deliverables
+
+- Our report is envisioned to contain multiple figures/quantitative results regarding: 
+  - Monthly liquidation profits for MakerDAO/Aave/Compound/dYdX.
+  - Understand the gas prices of liquidation events. Is there a way to avoid gas fee bidding contests in liquidations in an effort to preserve valuable block-space?
+  - Quantify the liquidation sensitivity (how much collateral is liquidated given a % price decline of the collateral) for MakerDAO/Aave/Compound/dYdX and most of the assets on these platforms.
+  - How many liquidation events are profitable to the liquidator?
```

Comment:
I would expect most?

Reply:
I guess MakerDAO is more risky, and we should see some losses.

Reply:
Would need a model to define profit, perhaps this could synergize with MEV taxonomy work/MEV vs. REV?

Reply:
Something like this?

Profit in USD = Revenue from liquidation - Value of repaid debt - Tx fees for liquidation

Reply:
One problem is that there are now professional liquidators who do not immediately sell the collateral claimed from liquidations and hold on to it for longer. For instance, 1inch's Aave v2 liquidation scheme would make such analysis quite hard without eliding them from the data set; yet, they're a huge venue for exiting liquidation trades. 

I would spend a lot of time trying to be very clear about the data cleaning methodologies when trying to do this analysis